server/models/anoynmous.js

```diff
@@ -0,0 +1,63 @@
+const db = require("../db/config");
+const SQL = require("sql-template-strings");
+
+// const Anonymous = require("./");
+
+class Anonymous {
+  constructor(data) {
+    this.id = data.id;
+    this.title = data.title;
+    this.pseudonym = data.pseudonym;
+    this.body = data.body;
+  }
+
+  static get all() {
+    return new Promise(async (resolve, reject) => {
+      try {
+        const anonymousData = await db.run(SQL`SELECT * FROM anonymousposts ;`);
+        const anoynmous = anonymousData.rows.map((d) => new anonymous(d));
+        resolve(anonymous);
+      } catch (err) {
+        reject("Error retrieving data");
+      }
+    });
+  }
+  // static findBytitle (title) {
```

Comment:
Nice data modelling and delete method!

Reply:
Also noticed some errors in lines 17/18  typo in 18/19 of "anonymous" and the new keyword might have to be used with "Anonymous" rather than "anonymous"